src/components/Button.jsx

```diff
@@ -0,0 +1,8 @@
+/* eslint-disable quotes */
+import React from "react";
+
+function Button() {
+  return <h1>should accept button as prop </h1>;
+}
```

Comment:
- [ ] The button component is expected to accept a name prop and the prop should be checked with prop-types with a value of "string".